eisbuk-admin/src/enums/routes.ts

```diff
@@ -5,6 +5,12 @@ export enum Routes {
   Debug = "/debug",
 }
 
+export enum CustomerRoute {
+  Calendar = "calendar",
+  BookIce = "book_ice",
+  BookOffIce = "book_off_ice",
+}
+
```

Comment:
I've named these "routes" even though they're views currently but will be used a routes later when we implement that logic. This is also to keep things consistent across views, path routes (in the future) and translations.